test/.eslintrc

```diff
@@ -2,13 +2,9 @@
 // lexically bound "this" prevents access to the Mocha test context.
 // See https://mochajs.org/#arrow-functions
 {
-  "env": {
-    "mocha": true
```

Comment:
I can see how this would be misleading with BigTest's own `@bigtest/mocha`.  Is there a preference for using eslint `/* global */` comments over this setting where a standard Mocha environment does apply?

Reply:
I prefer to be explicit with globals similar to treating them as imports. But since those webpack tests are started directly with mocha, I think it's fine to set the mocha env for those tests so I added a `.eslintrc` file to the `test/webpack` directory to re-enable it